website/docs/dbt-cli/configure-your-profile.md

```diff
@@ -1,12 +1,15 @@
 ---
-title: "Configure your profile"
+title: "Configuring your profile"
+description: "Configure your profile using the command line."
 ---
 
 ## Related documentation
-* [`profiles.yml` reference](/reference/profiles.yml)
 
-## How to connect to your warehouse when using the CLI
-When you invoke dbt from the CLI, dbt parses your `dbt_project.yml` for the name of the `profile` to use to connect to your data warehouse.
+* [`profiles.yml` reference](/reference/profiles.yml): Learn more about profile configuration.
```

Comment:
What do you think about identifying references so people can understand what they will be cliking?

Reply:
yes absolutely!